vcpkg/reference/vcpkg-json.md

```diff
@@ -408,6 +409,13 @@ The supported platform and build configurations for the [feature](../users/manif
 
 This field documents the platform configurations where the feature will build and run successfully.
 
+### <a name="feature-license"></a> [Feature][]: `"license"`
+
+The SPDX short license expression of the feature. A string or null. Optional. 
+If not provided, the license is the same as specified in the top level [license](#license) field.
+
+For more information see the top level [license](#license) field.
+
```

Comment:
I don't think the "For more information" line is needed since the link is already provided just before.
```suggestion
If not provided, the license is the same as specified in the top level [license](#license) field.

```